src/utils/staking/index.ts

```diff
@@ -43,3 +63,133 @@ export const buildStakingOutput = (
   }
   return psbtOutputs;
 };
+
+/**
+ * Get the staking term, it will ignore the `stakingTimeBlocks` and
+ * use the value from params if the min and max staking time blocks are the same
+ *
+ * @param {Params} params - The staking parameters.
+ * @param {number} term - The staking term.
+ * @returns {number} - The staking term.
+ */
+export const getStakingTerm = (params: Params, term: number) => {
```

Comment:
I think the existence of this function is confusing, as mentioned in another comment, we can just check whether the term is within the bounds of the parameters and throw an error otherwise. The caller of the library is responsible for passing a correct combination of parameters and staking term.